lib/resque_bus/adapter.rb

```diff
@@ -11,16 +12,48 @@ def enabled!
         QueueBus::Worker.extend(::QueueBus::Adapters::Resque::RetryHandlers)
       end
 
-      def redis(&block)
+      def redis=(server)
+        @queuebus_redis = nil
+        @queuebus_redis_url = server
+      end
+
+      def queuebus_redis
+        @queuebus_redis ||
+          if @queuebus_redis_url
```

Comment:
My original implementation involved copying the entire definition into this file.  I felt that doing so was brittle, because upstream changes to Resque would violate the principle of least surprise here.

Restricting URLs to `redis://` format is also surprising, because it's just resque, but we have different restrictions on how we specify Redis servers.

How would you feel about me adding a comment here explaining the code?  Would that be enough to ameliorate your concern?